other-docs/guides/getting-help-with-altis.md

```diff
@@ -8,72 +8,66 @@ Support happens via the Altis Dashboard. From your Altis Dashboard you will be a
 
 To open a support request, first you'll have to [login to the Altis Dashboard](dashboard.altis-dxp.com/).
 
-From there, you'll find the link to the support page in the top right corner. See below.
+From there, you'll find the link to the support page in the bottom left corner of the screen. See below.
 
 ![Altis Dashboard support link](./assets/altis-support-button.png)
 
-From this screen you'll be able to view your open issues, all previous issues, and you can also query open and previous issues if you're looking for something specific. 
+From the support panel screen you'll be able to view your open issues, all previous issues, and you can also query open and previously closed issues. 
 
 To open a new support ticket, simply click the blue 'Create New Ticket' button.
 
 ## Create Support Ticket
 
-A quick overview of the of the screen: 
+### Selecting a support issue type
 
-**Application**
-
-Select the specific Application or select all Application(s)/Environment(s) to which the support ticket concerns.
-
-**Type**
+To help guide you to finding the right kind of help, there's four categories a support ticket can fall under. They all have their own labels, but we'll go over them quickly.
 
-The type of support ticket. Currently there are four ticket types: Question, Incident, Problem, and Task. We'll go into more detail about which these categories below, but most tickets will generally fall into one of these four. 
+**Report an Incident**
 
-**Priority**
+Incidents are unplanned events that negatively impact service, or the quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if there is an issue which is causing an immediate business impact, this would be an Incident, and you can make a support request to help investigate.
 
-You can change the priority of a support issue if you need to. In most cases you shouldn't need to change this, but feel free to let us know if something is a priority.
+From the support team side, Incidents get a rapid response. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
+If the site is down for example, select 'Report an Incident'.
 
-**Subject**
+**Report an ongoing issue**
 
-Short description of the support request.
+Issues, also known as 'Problems' are the cause of, or can be the potential cause of future Incidents. If you're encountering a problem which could cause potential Incidents, or if you've run into a bug, select 'Report an ongoing issue'.
 
-**Description**
 
-The body of the support request.
+**Request an infrastucture change**
 
-**CCs**
+There’s an array of tasks that you might need help with. Typically, this might be the provisioning of an additional environemnent, or it could be to run an operation that requires a cloud engineer, upgrading PHP for example.
```

Comment:
```suggestion
There’s an array of tasks that you might need help with. Typically, this might be the provisioning of an additional environment, or it could be an operation that requires a cloud engineer to run, such as upgrading the PHP version.
```